lib/base/scriptutils.cpp

```diff
@@ -30,9 +30,12 @@
 #include <boost/regex.hpp>
 #include <algorithm>
 #include <set>
+
+#ifdef ICINGA2_WITH_MSI_FUNCTION
```

Comment:
Is this really necessary? It looks like Nano would require different Windows binaries on its own. Is there a built-in define we could refer to, similar to _WIN32?